spring-integration-core/src/test/java/org/springframework/integration/resource/ResourceInboundChannelAdapterParserTests.java

```diff
@@ -82,9 +83,10 @@ public void testDefaultConfig() {
 		context.close();
 	}
 
-	@Test(expected = BeanCreationException.class)
+	@Test
 	public void testDefaultConfigNoLocationPattern() {
-		new ClassPathXmlApplicationContext("ResourcePatternResolver-config-fail.xml", getClass()).close();
+		assertThatThrownBy(() -> new ClassPathXmlApplicationContext("ResourcePatternResolver-config-fail.xml", getClass()).close())
```

Comment:
I think with this AssertJ API we don't need that `close()` any more.
Previously it was a code quality question to have `Closable` resource closed.
The fact that it ended up with an exception was not visible for that tool.